native/cocos/platform/android/AndroidPlatform.cpp

```diff
@@ -61,9 +62,24 @@
     }
 
 #define INPUT_ACTION_COUNT 6
+#define COCOS_EXIT_GAME 101
 
 extern int cocos_main(int argc, const char **argv); // NOLINT(readability-identifier-naming)
 
+extern "C" {
+bool perform_in_game_thread(struct android_app *app, cc::AndroidPlatform::GameThreadFunc func) {
```

Comment:
Where is `perform_in_game_thread` invoked?

Reply:
invoke from here:  https://github.com/cocos/cocos-engine-external/pull/253